### PR TITLE
fix(pulse):Adding pulse post for Lf modems

### PR DESCRIPTION
**Pull request by AnneThKu**

### CODE REVIEW

**Comment by AnneThKu** [comment]
Screenshot 2021-10-12 at 2 00 57 PM

File: _posts/2021-10-12-Subnero-Long-Range-Modems.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <p>WNC-L5LSS3 (Standalone Configuration)</p>
+    </div>
+    <div style="flex:50%;">
+    <img src="{{site.baseurl}}/images/thumbnail-wnc-l5lss4.png" style='width:60%;border:1px solid #555'>
+    <p>WNC-L5LSE3 (Embedded Configuration)</p>
+</div>
+</div>
+<div class="spacing"></div>
+
+Subnero’s [L5L series](https://subnero.com/products/modem.html?&section=L5Lseries) long-range modems provide the capability to communicate underwater over 20 km in some of the most challenging underwater environments with data rates up to 100 bps, in a compact form factor.
```
Comment (manuignatius) [inline _posts/2021-10-12-Subnero-Long-Range-Modems.md]:
```suggestion
We are excited to launch the new long-range modems. The [L5L series](https://subnero.com/products/modem.html?&section=L5Lseries) long-range modems provide the capability to communicate underwater over 20 km in some of the most challenging underwater environments with data rates up to 100 bps, in a compact form factor.
```

File: _posts/2021-10-12-Subnero-Long-Range-Modems.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## APPLICATION DIVERSITY
+Ideal for applications such as passive acoustic monitoring, water quality monitoring, long range AUV command and control, seismic monitoring.
+
+## SOFTWARE-DEFINED CORE
+Users can fully configure the modulation and FEC parameters along with support for arbitrary waveform transmission and recording, through UnetStack APIs.
+
+## SEAMLESS INTEGRATION
+L5L series modems supports seamless integration with Subnero's M25M series modems to build true heterogenous networks.
+
+We hope you find our long range modems exciting.
```
Comment (manuignatius) [inline _posts/2021-10-12-Subnero-Long-Range-Modems.md]:
```suggestion

```

File: _posts/2021-10-12-Subnero-Long-Range-Modems.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+We love to hear from you. Let us know your thoughts and comments through info@subnero.com.
```
Comment (manuignatius) [inline _posts/2021-10-12-Subnero-Long-Range-Modems.md]:
```suggestion
Contact us at info@subnero.com for more details.
```

File: _posts/2021-10-12-Subnero-Long-Range-Modems.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Introducing Subnero Long Range Modems
+banner : images/banner-multichannel.jpg
```
Comment (manuignatius) [inline _posts/2021-10-12-Subnero-Long-Range-Modems.md]:
```suggestion
banner : images/banner-silver-l5l.jpg
```

**Review by manuignatius — CHANGES_REQUESTED** [review]

**Review by manuignatius — APPROVED** [review]

**Comment by Florie25** [comment]
Looks good to me

**Pull request merged**